### ISSUE TITLE
Iteration through pages

### ISSUE THREAD

**Issue by flaclimate** [asjolaviral/scopusapi#2]
Thank you for fixing the previous issue.
As far as I understand, the new code shoudl also iterate the query beyond the first 25 entries (first page).  However, it seems to me that is not happening. Am I doing anything wrong perhaps? Thanks!

**Comment by asjolaviral** [comment]
Yes. the default is 25 records, but scopus provides maximum 200 results.
URL: https://dev.elsevier.com/api_key_settings.html
